samples/python/rag/README.md

```diff
@@ -0,0 +1,33 @@
+# Retrieval Augmented Generation Sample
+
+This example showcases inference of Text Embedding Models. The application limited configuration configuration options to encourage the reader to explore and modify the source code. For example, change the device for inference to GPU. The sample features `openvino_genai.TextEmbeddingPipeline` and uses text as an input source.
```

Comment:
Please add use case description, why would someone need Text Embedding Models and that it's only a part of bigger potential pipeline